xblock/reference/plugins.py

```diff
@@ -171,3 +130,42 @@ def __set__(self, xblock, value):
         We don't support this until we figure out what this means
```

Comment:
In this case, the distinction is simple: 

* We are moving the field type to be a supported, stable API. People can rely on this API to continue to exist.
* We are not moving the service to be a supported, stable API. That means we may change it in the future if we come up with a better way to do things. If we do change this, the field type will continue to work, since we control both ends. 
* Both are currently considered to have robust, stable implementations. It's just that we are only committing to one of them as a long-term contract. If XBlocks use the fs field type, they will continue to work for whatever support timeframe we end up having on the current API, and with similar deprecation mechansims. If XBlocks use the service, they may break at any time as we evolve the API. 

Again, it would be helpful if you actually took the time to read the references I already gave you several times. At the very least, you should take the time to actually understand the code *before* commenting further. The PR mechanism is intended as a way to comment on specific code, not to request a free programming tutorial.